### PR TITLE
[UI-3]: Update design on login/register page

### PR DESCRIPTION
**Pull request by Yuliasik**

### CODE REVIEW

File: time-manager-ui/src/app/structure/auth/login-form/login-form.component.ts [hunk 1]
```diff
@@ -44,19 +44,23 @@ export class LoginFormComponent implements OnInit {
...
-          this.router.navigate(['']);
-        } else {
-          alert("Register failed!")
-        }
-      })
+  back() {
+    this.router.navigate(['/greeting']);
   }
 
+  // register() {
```
Comment (Ivanochko) [inline time-manager-ui/src/app/structure/auth/login-form/login-form.component.ts:51]: Please remove these comments

File: time-manager-ui/src/app/structure/auth/register/register.component.ts [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+          this.dialogConfig.data = {
+            message: errorRes.error
+          }
+          this.dialog.open(DialogComponent, this.dialogConfig);
+        }
+      )
+  }
+
+  isDifferentPassword(): boolean {
+    return (this.password && this.repeatPassword) ? this.password !== this.repeatPassword : false
```
Comment (Ivanochko) [inline time-manager-ui/src/app/structure/auth/register/register.component.ts:73]: What about:
`return this.password && this.repeatPassword && this.password !== this.repeatPassword`

File: time-manager-ui/src/app/structure/auth/login-form/login-form.component.ts (hunk 1 above)
Comment (Yuliasik) [reply time-manager-ui/src/app/structure/auth/login-form/login-form.component.ts]: Thanks,  I remove it.

File: time-manager-ui/src/app/structure/auth/register/register.component.ts (hunk 2 above)
Comment (Yuliasik) [reply time-manager-ui/src/app/structure/auth/register/register.component.ts]: It is pretty simple. Thank you

**Review by Ivanochko — APPROVED** [review]

**Pull request merged**